deploy/task-cad-checks.yaml

```diff
@@ -21,7 +21,7 @@ spec:
           file=${folder}/payload.json
           echo '$(params.payload)' > $file
           # run the cadctl command
-          RESULT=$(cadctl cluster-missing --payload-path $file )
+          cadctl cluster-missing --payload-path $file
```

Comment:
The reason this was wrapped in a RESULT, was to pipe the output to another task for processing..
We should also have the json output saved to a file and passed as an output for future plumbing

Reply:
This other task does not exist as of now, right?

The problem here is that due to the RESULT variable the stdout gets never forwarded to stdout. The variable catches it and this is why we don't see stdout messages in the container logs.

Can we do both some how? Printing and inserting into a file? The `tee` command should work this way, right?